### PR TITLE
RAB-1239: CI workflow for 7.0

### PR DESCRIPTION
**Pull request by ghoussard**
**Description (for Contributor and Core Developer)**

I updated Circle CI config for the 7.0 branch.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: .circleci/jobs/build_and_deploy.yml [hunk 1]
```diff
@@ -101,13 +101,6 @@ jobs:
                   name: Require proper dev CE branch
                   command: |
                       sed -i "s#akeneo/pim-community-dev\": \"7.0.x-dev@dev#akeneo/pim-community-dev\": \"dev-${CIRCLE_BRANCH}#" composer.json
-                      sed -i "s#akeneo/pim-community-dev\": \"7.0.x-dev@dev#akeneo/pim-community-dev\": \"dev-${CIRCLE_BRANCH}#" grth/composer.json
-            - run:
-                  name: Require proper dev onboarder bundle branch
```
Comment (ghoussard) [inline .circleci/jobs/build_and_deploy.yml:106]: Useless step (work already done by "Update composer.json if same branch exists in pim-onboader for EE" step)

**Review by tseho — APPROVED** [review]

**Review by MadeWilson — APPROVED** [review]

File: .circleci/workflows/editions/community.yml [hunk 2]
```diff
@@ -46,47 +46,3 @@ workflows:
                   <<: *slack-fail-post-step
```
Comment (StevenVAIDIE) [inline .circleci/workflows/editions/community.yml:46]: We don't want to have a notification if step fail for tagged version
```suggestion
```

Can you remove those lines ?

File: .circleci/jobs/build_and_deploy.yml [hunk 3]
```diff
@@ -101,13 +101,6 @@ jobs:
...
-                  command: |
-                      export GIT_SSH_COMMAND='ssh -i ~/.ssh/id_rsa_1f25f8bb595295f6e2f2972f30d4e966 -o UserKnownHostsFile=~/.ssh/known_hosts -o IdentitiesOnly=Yes'
-                      ONBOARDER_BRANCH=$(if [[ -z $(git ls-remote --exit-code --heads git@github.com:akeneo/pim-onboarder.git ${CIRCLE_BRANCH}) ]]; then echo "master"; else echo ${CIRCLE_BRANCH}; fi)
-                      sed -i "s#akeneo/pim-onboarder\": \"^5.0.9#akeneo/pim-onboarder\": \"dev-${ONBOARDER_BRANCH}#" composer.json
```
Comment (StevenVAIDIE) [inline .circleci/jobs/build_and_deploy.yml:110]: Can you remove the following jobs: build_prod, test_deploy, deploy_pr_environment, delete_pr_environment, build_docker_images

**Review by StevenVAIDIE — COMMENTED** [review]
Like EE PR, do not forget to undo change you made on Circleci Trigger => https://app.circleci.com/settings/project/github/akeneo/pim-community-dev/triggers/c51783cc-ca8a-4e92-b44e-93b2ca834b56?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fakeneo%2Fpim-community-dev&triggerSource=

**Review by StevenVAIDIE — APPROVED** [review]

File: .circleci/jobs/build_and_deploy.yml (hunk 3 above)
Comment (ghoussard) [reply .circleci/jobs/build_and_deploy.yml:110]: Done

**Pull request merged**